code.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+
+def lambda_handler(event, context):
+    if "body" not in event:
+        return "no body found"
```

Comment:
> [!CAUTION]
> The function returns a string instead of a proper HTTP response, which is required for AWS Lambda functions that are triggered by API Gateway. This will cause the API Gateway to receive an incorrect response format.

> [!TIP]
> Return a properly formatted HTTP response object with status code, headers, and body.

```python
if "body" not in event:
    return {
        "statusCode": 400,
        "body": json.dumps({"message": "no body found"})
    }
```